tests/Shared/LayoutRenderers/W3CLoggerLayoutTests.cs

```diff
@@ -49,6 +49,7 @@ public void W3CLoggerLayoutNoContextTest()
             Assert.Equal(expectedBody, body);
         }
 
+        /*
```

Comment:
Can this be removed again ?

Reply:
probably, I will try

Reply:
I will uncomment it.  i have seen both unit tests in that file fail randomly upon making commits to NLog.Web.  But they never fail on my own laptop.